src/components/common/PeopleAndRoles.vue

```diff
@@ -184,6 +184,7 @@
         <span>Add the Completing Party</span>
       </v-btn>
 
+      <!-- *** FUTURE: don't show Add a Person for short-form amalgamation -->
```

Comment:
I'll keep working on some of the FUTURE things in this ticket/PR, or maybe a future PR if we need this code merged soon.